src/StatusQ/Popups/StatusSearchPopup.qml

```diff
@@ -0,0 +1,192 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.12
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+
+
+StatusModal {
+    id: root
+    width: 700
+    height: (!!searchResults && (searchResults.count >= 0) && (searchText !== "")) ? (((view.count < 5)) ? 560 : 770) : 142 //970
+    anchors.centerIn: parent
+    property string searchText: inputText.input.text
+    property string selectionBadgeImage: ""
+    property string selectionBadgePrimaryText: ""
+    property string selectionBadgeSecondaryText: ""
+    property string selectionBadgeIdenticonColor: ""
+    property bool loading
+    property Component searchOptionsPopupMenu
+    function resetSelectionBadge() {
+        selectionBadgeImage = "";
+        selectionBadgePrimaryText = "Anywhere";
+        selectionBadgeSecondaryText = "";
+        selectionBadgeIdenticonColor = "";
+    }
+    property ListModel searchResults: ListModel {}
+
+    contentItem: Item {
+        width: parent.width
+        height: parent.height
+        ColumnLayout {
+            id: contentItemColumn
+            anchors.fill: parent
+            spacing: 0
+            Item {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 63
+                StatusInput {
+                    id: inputText
+                    anchors.left: parent.left
+                    anchors.leftMargin: 8
+                    anchors.right: parent.right
+                    anchors.rightMargin: 8
+                    anchors.bottom: parent.bottom
+                    input.icon.name: "search"
+                    input.placeholderText: "Search"
+                    input.bgVisible: false
+                }
+            }
+            StatusMenuSeparator { Layout.fillWidth: true }
```

Comment:
Semantically, this is not a menu, so we might want to consider introducing a general `StatusSeparator` in the near future.

Reply:
Can we just rename this one if there is no difference between those 2? Maybe create a task for the near future?

Reply:
Yea absolutely. Let's just keep this as is and rename this in the future.